### DISCUSSION TITLE
Lesson 1:  contract name creatin error

### DISCUSSION THREAD

**Discussion by GitMyHubby** [smartcontractkit/full-blockchain-solidity-course-py#929] [Q&A] [answered]
Hello all, it didn't take long to find the first obstacle in lesson 1 ahah 

I am trying to name the contract but when I compile, it gives me the error in the image. 

![Error](https://user-images.githubusercontent.com/98923680/152253494-e1a0e624-9e37-459f-b5d0-9a939e6a2ced.png)

**Comment by cromewar** [comment, marked as answer]
Hello @GitMyHubby you need to close the function with `{}`

**Reply by GitMyHubby** [reply]
Thank you!
